lib/show_menu.rb

```diff
@@ -0,0 +1,11 @@
+require_relative "menu"
+
+class ShowMenu
```

Comment:
I'm confused as to why this is a class, and why you have behaviour (menu.display_menu) in your initialize. Showing the menu is a behaviour so I would expect to see a show_menu method on another class, rather than a ShowMenu class itself. 

You might (and probably should) want to separate the concerns of storing the menu data and displaying the data, in which case you could have a Menu class, which has a list of items and a get_menu method, and a MenuPrinter class, which has a show_menu class, which (a) gets the menu from menu, (b) formats it, and (c) printers it to the terminal.

The key here is that each class name is a noun and each method name is a verb, and the responsibilities are clearly defined - Menu is concerned with the data and MenuPrinter is concerned with display.